docs/src/helpers/logo-helper.js

```diff
@@ -17,10 +17,18 @@
  */
 
 import BackpackLogoWhite from '../static/backpack-logo-white.svg';
+import BackpackLogoGrey from '../static/backpack-logo-sky-grey.svg';
 import BackpackLogoPride from '../static/backpack-logo-pride.svg';
+import BackpackLogoPrideGrey from '../static/backpack-logo-pride-sky-grey.svg';
 
-export const getBackpackLogo = () => {
+export const getBackpackLogo = (isHeroImage = false) => {
```

Comment:
Can we rename this to be more `why` and less `what`?
```suggestion
export const getBackpackLogo = (lightVariant = false) => {
```

Reply:
I think `lightVariant` might be confusing as it doesn't depend on if the site is using a dark theme/mode more it is for solid vs coloured backgrounds

Reply:
can we rename it `overImage` or something then, to capture that this is for use on top of images?

Reply:
Done!